### ISSUE TITLE
SwaV Cifar10 benchmark collate function arguments error

### ISSUE THREAD

**Issue by guarin** [lightly-ai/lightly#597]
The swav collate function in the benchmark is missing the `crop_min_scales` and `crop_max_scales` arguments

https://github.com/lightly-ai/lightly/blob/bc04a8bad92eb91fc7c2809506348df15d85b69a/docs/source/getting_started/benchmarks/cifar10_benchmark.py#L103-L106

It currently raises the following error:

```
ValueError: Length of crop_sizes and crop_min_scales must be equal but are 1 and 2. 
```

This is because the min and max scales are by default set to a list with length two but we only pass a single crop size.

**Comment by JeanKaddour** [comment]
did you find a fix for that?

**Comment by philippmwirth** [comment]
Hi @JeanKaddour, having crop sizes of `16` doesn't make a lot of sense because the ResNet will not be able to handle such small images. Instead it's probably better to simply change the other parameters as well, e.g.
```python
swav_collate_fn = lightly.data.SwaVCollateFunction(
    crop_sizes=[32],
    crop_counts=[2],
    crop_min_scales=[0.14],
    crop_max_scales=[1.0],
)
```

Alternatively, you can use larger input sizes:
```python
swav_collate_fn = lightly.data.SwaVCollateFunction(
    crop_sizes=[224, 96],
    crop_counts=[2, 4]
)
```

**Comment by JeanKaddour** [comment]
Thanks a lot for your help!

**Comment by philippmwirth** [comment]
Closed by #694
